### ISSUE TITLE
Hacks tab not working on Japan rom

### ISSUE THREAD

**Issue by XingYaoA** [SM64-TAS-ABC/STROOP#118]
The hacks tab not working on Japan rom.

**Comment by danebou** [comment]
Which hack, any of them? Also do you have pure interpretter enabled? Are you using mupen?

**Comment by XingYaoA** [comment]
@danebou Yes,all of them,pure interpreter is enabled and  I'm using mupen.
(Hacks tab Works on USA rom)
